t/storage.t

```diff
@@ -40,21 +53,21 @@ describe "HTFeed::Storage" => sub {
       my $volume = stage_volume(@_);
 
       my $storage = HTFeed::Storage::LocalPairtree->new(
-        volume => $volume);
+        volume => $volume,
+        config => config_for_class('HTFeed::Storage::LocalPairtree'));
```

Comment:
This needs to be untangled a little bit. I *think* we could probably change TempDirs to spit back a config that can be passed to this instead of needing to fuss with the global storage class config in `config_for_class`

Reply:
I was able to do this as long as I made sure to check whether `tempdir` had already been called for a given directory type. Still seems kind of brittle. Here's what I added:
```
sub config_for_storage_class {
  my $self  = shift;
  my $class = shift;

  my $tmpdir = $self->{tmpdir};
  my $config = {};

  foreach my $dirtype ($self->repo_dirtypes) {
    unless (exists $self->{$dirtype}) {
      $self->{$dirtype} = tempdir("$tmpdir/feed-test-$dirtype-XXXXXX");
    }
    $config->{$dirtype} = $self->{$dirtype};
  }
  return $config;
}
```

Reply:
That's probably OK, although worth verifying if there are actually cases where the tests rely on having the same temporary directories between two storage instances or not.